charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -188,23 +187,15 @@ spec:
         {{- end }}
 
         livenessProbe:
-          httpGet:
```

Comment:
Why it is removed at all? Let's leave these by defaqult and add possibility to pass these valyes with helm chart?

Reply:
https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/128#discussion_r1726750186